lib/Gitter/Client.php

```diff
@@ -61,13 +62,19 @@ public function getRepository($path)
         return new Repository($path, $this);
     }
 
+    /**
+     * @param $repository
+     * @param $command
+     *
+     * @return string
+     */
     public function run($repository, $command)
     {
         if (version_compare($this->getVersion(), '1.7.2', '>=')) {
             $command = '-c "color.ui"=false ' . $command;
         }
 
-        $process = new Process(ProcessUtils::escapeArgument($this->getPath()) . ' ' . $command, $repository->getPath());
+        $process = new Process($this->getPath() . ' ' . $command, $repository->getPath());
```

Comment:
Yeah, understood. The issue now is that `ProcessUtils::escapeArgument` is deprecated.
I believe that there are two changes that implement a similar fix.

Here: https://github.com/tomzx/gitter/pull/4/files#diff-fbbb944d95f0066ae1654a20caaa23a66c25407ee8e37e5b03ea23b6b28c25ceR118
And Here: https://github.com/tomzx/gitter/pull/4/files#diff-91c8b3868401dfbe046f349d9f7e67fe88f8f9c9e00be06198655155da98cbc7R237

I'll do some testing on windows though to make sure those covered our bases on that issue.

Reply:
OK - so after some testing I think that removing this is fine.

From what I'm seeing either the two fixes above - merged from upstream - address the windows paths issue. Or potentially, the `symfony/console:^4.0` branch accounts for it internally? Perhaps a little bit of A and B?

# Testing
---
Builds were made using PHP 7.2 on a mac system via box. I made three builds then used a USB to execute tests via Windows PHP 7.2 build from php.net.

- php-semver-checker-git.new.phar - the "default" version for this PR.
- php-semver-checker-git.backport.phar - a version that adds a hacky helper file to test backporting the method.
- php-semver-checker-git.array.phar - a version that uses an "array" which allows `symfony/process` to build the command.

# Results
---
| build version | Nix Status | Windows Status |
|---------------|------------|----------------|
| new (default) | Works      | Works          |
| backport      | Works      | Works          |
| array         | broke AF   | broke AF       |

Based on the results I think that the backport isn't needed and this PR (as is) should be ok.
However it dawns on me that the https://github.com/tomzx/php-semver-checker-git/issues/29 patch may need to be tested in source, rather than Phar compile to find the Windows issue?

For instance would it be better for me to check out the repo and run unit tests on windows with https://github.com/tomzx/php-semver-checker-git on the various branches?